src/CaptureFile/include/CaptureFile/CaptrureSectionInputStream.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_FILE_MAIN_SECTION_INPUT_STREAM_H_
```

Comment:
Adjust to `CAPTURE_FILE_CAPTURE_SECTION_INPUT_STREAM_H_`.

Reply:
done.